packages/boilerplates/BoilerplatePackageJson.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export type BoilerplatePackageJson =  {
```

Comment:
Json is not an important information, I would drop this postfix. It is a BoilerplatePackage - where its sourcing from can be anything